test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -133,6 +133,56 @@ test.describe(
 				)
 			).toBeVisible();
 		} );
+
+		test( 'default to my most recently created menu', async ( {
+			admin,
+			page,
+			editor,
+			requestUtils,
+		} ) => {
+			await admin.createNewPost();
+			await requestUtils.createNavigationMenu( {
+				title: 'Test Menu 1',
+				content:
+					'<!-- wp:navigation-link {"label":"WordPress","type":"custom","url":"http://www.wordpress.org/","kind":"custom","isTopLevelLink":true} /-->',
+			} );
+
+			// The two menus are created with the same timestamp, so we need to wait a bit to ensure the second one is created after the first one.
```

Comment:
```suggestion
			// Pause before creating the 2nd menu to ensure that each menu has a different timestamp. Failure
			// to do this would result in two menus with identical "created" timestamps.
```

Reply:
I removed this line because the recent refactor made it unnecesary